src/commands/createNewProject/ProjectCreateStep/TypeScriptProjectCreateStep.ts

```diff
@@ -42,24 +42,31 @@ export class TypeScriptProjectCreateStep extends JavaScriptProjectCreateStep {
     protected getPackageJsonDevDeps(context: IProjectWizardContext): { [key: string]: string } {
         // NOTE: Types package matches node worker version, not func host version
         // See version matrix here: https://www.npmjs.com/package/@azure/functions
-        let nodeWorkerVersion: string;
+        let funcTypesVersion: string;
+        // For the node types package, we'll use the latest LTS version possible
+        // See version matrix here: https://docs.microsoft.com/azure/azure-functions/functions-versions?pivots=programming-language-javascript#languages
+        let nodeTypesVersion: string;
         switch (context.version) {
             case FuncVersion.v4:
-                nodeWorkerVersion = '3';
+                funcTypesVersion = '3';
+                nodeTypesVersion = '16';
                 break;
             case FuncVersion.v3:
-                nodeWorkerVersion = '2';
+                funcTypesVersion = '2';
+                nodeTypesVersion = '14';
                 break;
             case FuncVersion.v2:
-                nodeWorkerVersion = '1';
+                funcTypesVersion = '1';
+                nodeTypesVersion = '10';
```

Comment:
😨

What's the support state for Functions v2?

Reply:
https://docs.microsoft.com/en-us/azure/azure-functions/language-support-policy#retirement-policy-exceptions

30 September 2022 is the listed date to end support for Node 10 (the latest version for v2). Users will still be able to run Node 10, but this is the main difference:
> If you're running functions apps using an unsupported language version, you'll be required to upgrade before receiving support for the function apps.

Aka my team doesn't have to respond to incidents until user's upgrade